webxr-api/frame.rs

```diff
@@ -26,6 +26,9 @@ pub struct Frame {
 
     /// Events that occur with the frame.
     pub events: Vec<FrameUpdateEvent>,
+
+    /// Value of time::precise_time_ns() when frame was obtained
+    pub time_ns: u64,
```

Comment:
Use an `Instant` here?

Reply:
Instant can only be compared with other Instants, and we don't use that in servo for `Performance.now()`